scalafix/rules/src/main/scala/fix/Stable212Base.scala

```diff
@@ -49,7 +49,252 @@ trait Stable212Base extends CrossCompatibility { self: SemanticRule =>
     "_root_.scala.collection.Iterable#"
   )
 
+  object Breakout {
+    implicit class RichSymbol(val symbol: Symbol) {
+      def exact(tree: Tree)(implicit index: SemanticdbIndex): Boolean =
+        index.symbol(tree).fold(false)(_ == symbol)
+    }
+
+    val breakOut = SymbolMatcher.exact(Symbol("_root_.scala.collection.package.breakOut(Lscala/collection/generic/CanBuildFrom;)Lscala/collection/generic/CanBuildFrom;."))
+
+    // infix operators
+
+    val `TraversableLike ++`  = Symbol("_root_.scala.collection.TraversableLike#`++`(Lscala/collection/GenTraversableOnce;Lscala/collection/generic/CanBuildFrom;)Ljava/lang/Object;.")
+    val `Vector ++`           = Symbol("_root_.scala.collection.immutable.Vector#`++`(Lscala/collection/GenTraversableOnce;Lscala/collection/generic/CanBuildFrom;)Ljava/lang/Object;.")
+    val `List ++`             = Symbol("_root_.scala.collection.immutable.List#`++`(Lscala/collection/GenTraversableOnce;Lscala/collection/generic/CanBuildFrom;)Ljava/lang/Object;.")
```

Comment:
This should only be TraversableLike. The way symbols work in scalafix right now is similar to go-to-definition in an IDE, It will hit the most concrete class first. @olafurpg Is there a way only to hit the base class?